signalbot/bot.py

```diff
@@ -306,6 +309,35 @@ def _is_valid_uuid(self, receiver_uuid: str):
         except ValueError:
             return False
 
+    def _is_username(self, receiver_username: str):
```

Comment:
Would you be willing to add some unit testing for this function? It should be done in the the `test_bot.py` file, adding a new class that tests the bot send function.

Reply:
I will get around to it

Reply:
For now added tests only for the `_is_username` method. Not sure exactly how send function should be tested correctly with group ids and phone numbers. If the tests are going to be updated, feel free to mention me and I will refactor according to how sending to phone number and group_id receiver is done.

Reply:
Looks great, thanks! Just one last one, can you refactor it to do it in a for loop to avoid code duplication?
```python
        valid_names = [
            "Usr.99",
            ...
            "usernameeeeeeeeeeeeeeeeeeeeeeeee.999999999",
        ]
        for name in valid_names:
            self.assertTrue(self.signal_bot._is_username(name))
```

Reply:
Code review fixes applied

Reply:
Awesome. Sadly, it looks like the pre-commit checks failed. You just need to add commas at the end of the list. The recommended way is to have the pre-commit hook installed locally, so that it runs the checks on every commit, like so https://github.com/filipre/signalbot?tab=readme-ov-file#local-development-and-package

Reply:
Yeah, forgot to run black. Should be fine now.